draft-irtf-cfrg-hpke.md

```diff
@@ -679,26 +681,26 @@ discussion.
 HPKE allows multiple encryption operations to be done based on a
 given setup transaction.  Since the public-key operations involved
 in setup are typically more expensive than symmetric encryption or
-decryption, this allows applications to "amortize" the cost of the
+decryption, this allows applications to amortize the cost of the
 public-key operations, reducing the overall overhead.
 
 In order to avoid nonce reuse, however, this decryption must be
 stateful. Each of the setup procedures above produces a context object
 that stores the required state:
 
-* The AEAD algorithm in use
-* The key to be used with the AEAD algorithm
-* A base nonce value
-* A sequence number (initially 0)
-* An exporter secret (see {{hpke-export}})
+* The AEAD algorithm and corresponding key, an initial nonce value,
```

Comment:
Disagree with collapsing all of these bullets.  If you want to group them, you could have top-level "AEAD parameters" and "Export parameters"

Reply:
It felt cleaner, but I don't feel strongly about this. I made a slight variation that's closer to what we had.